src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -806,32 +804,36 @@ static void glb_rr_connectivity_changed_locked(grpc_exec_ctx *exec_ctx,
                                                void *arg, grpc_error *error) {
   rr_connectivity_data *rr_connectivity = arg;
   glb_lb_policy *glb_policy = rr_connectivity->glb_policy;
-
   const bool shutting_down = glb_policy->shutting_down;
```

Comment:
Probably no need for this variable anymore, since the value is only used in one place (on the next line).